adcircpy/driver.py

```diff
@@ -11,24 +11,24 @@
 import numpy as np
 from psutil import cpu_count
 
-from adcircpy.mesh import AdcircMesh
 from adcircpy._fort15 import _Fort15
 from adcircpy.forcing import Tides  # , Winds
+from adcircpy.mesh import AdcircMesh
 from adcircpy.outputs.collection import OutputCollection
-from adcircpy.server._driver_file import _DriverFile
 from adcircpy.server import SSHConfig, SlurmConfig
+from adcircpy.server._driver_file import _DriverFile
 
 
 class AdcircRun(_Fort15):
 
     def __init__(
-        self,
-        mesh: AdcircMesh,
-        start_date: datetime,
-        end_date: datetime,
-        spinup_time: timedelta = None,
-        netcdf: bool = True,
-        server_config: Union[int, SSHConfig, SlurmConfig] = None
+            self,
```

Comment:
Why the extra identation?

Reply:
from what I've seen in PEP8 [it wants extra indentation blocks in continuation and signatures](https://www.python.org/dev/peps/pep-0008/#indentation)

Reply:
Ok, thanks for the clarification!